### PR TITLE
Fixes #2018 Add CSV module

### LINKED ISSUE #2018: Add CSV module
Support CSV files using Apache commons-csv library

xref: https://issues.redhat.com/browse/ENTESB-13746

### PR DESCRIPTION
**Pull request by rkorytkowski**

### CODE REVIEW

**Comment by rkorytkowski** [comment]
@igarashitm please have a look. 

From what I see I still need to add a component test for Camel and Camel3. Any important tests I missed in terms of adding a module?

Does it make sense to enable it in standalone?

I will be also adding more business logic tests...

File: lib/modules/dfdl/module/src/main/java/io/atlasmap/dfdl/module/DfdlModule.java [hunk 1]
```diff
@@ -1,132 +0,0 @@
-/**
```
Comment (igarashitm) [inline lib/modules/dfdl/module/src/main/java/io/atlasmap/dfdl/module/DfdlModule.java:1]: Removing DFDL module? it's there not only for CSV - CSV is just an example.

**Comment by igarashitm** [comment]
@rkorytkowski other than DFDL module LGTM! It looks to me camel test is not mandatory as it doesn't change something specific to camel component. The itests will do the job enough I guess?
Also can you tweak commit message, to have `feat:` header and `Fixes: #2018` at second line?
https://github.com/atlasmap/atlasmap/wiki/AtlasMap-Developer-Memo#commit-message

File: lib/modules/dfdl/module/pom.xml [hunk 2]
```diff
@@ -66,6 +66,12 @@
       <artifactId>atlas-core</artifactId>
       <classifier>tests</classifier>
     </dependency>
+    <dependency>
```
Comment (igarashitm) [inline lib/modules/dfdl/module/pom.xml:69]: remove this dependency?

**Comment by rkorytkowski** [comment]
@igarashitm, thanks. Updated the commit message.

**Comment by igarashitm** [comment]
Sorry to be scattering, we'd need to add csv module dependency to the camel/camel3/standalone/standalone-reactui POM.

**Comment by rkorytkowski** [comment]
Good catch. Fixed camel/camel3/standalone/standalone-reactui POM.

**Pull request merged**